packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/ConceptResource.java

```diff
@@ -0,0 +1,74 @@
+package software.uncharted.terarium.hmiserver.resources.dataservice;
+
+import io.quarkus.security.Authenticated;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import software.uncharted.terarium.hmiserver.models.dataservice.Concept;
+import software.uncharted.terarium.hmiserver.proxies.dataservice.ConceptProxy;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/api/concepts")
+@Authenticated
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name = "Concept REST Endpoints")
+public class ConceptResource {
+
+	@Inject
+	@RestClient
+	ConceptProxy proxy;
+
+	@GET
+	public Response searchConcept(
+		@QueryParam("term") final String term,
```

Comment:
`@QueryParam` parameters are optional fields will pass in `null` if not defined.  Do we have guarantees that `proxy.searchConcept` can accept `null` values?  If not we may want to set these using `@DefaultValue`

Reply:
This question holds for all of the XYZResource.java classes in the dataservice directory -  I won't spam on each of them.

Reply:
the default values for ints and strings should be valid for these query params